v2/cf-nodejs-logging-support/src/lib/logger/logger.ts

```diff
@@ -46,10 +50,12 @@ export default class Logger {
     }
 
     logMessage(levelName: string, ..._args: any) {
-        if (!this.isLoggingLevel(levelName)) return;
+        if (!this.isLoggingLevel(levelName) || this.loggingLevelThreshold == Level.OFF) return;
```

Comment:
Why is the second check required? I think `isLoggingLevel` should already cover `Level.OFF`.

Reply:
I think it is necessary because if both the logging level treshold and the log level are `Level.OFF`, then `isLoggingLevel` would return `true` and the message would be logged. However this should not be the case, if log level is `Level.OFF` then the log should be ignored regardless of the treshold.

Reply:
The `Level.OFF` check is now covered in the method `isLevelEnabled`